src/createAdder.js

```diff
@@ -20,7 +20,14 @@
  * @param {number} initialValue
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let adder = initialValue;
+
+  return function() {
+    adder = [...arguments]
+      .reduce((first, second) => first + second, adder);
```

Comment:
редьюс не меняет оригинальный массив
```suggestion
    adder = arguments.reduce((first, second) => first + second, adder);
```